### PR TITLE
minimum schema version updated

### PR DESCRIPTION
**Pull request by gcase555**
The goal of this pull request is to update the min schema version to something closer to current than 6(which is way behind).

The integration tests broke in `master-integration` but will likely not work until I merge in this change and add them back to the travis builds.

### CODE REVIEW

File: pkg/storage/schema.go [hunk 1]
```diff
@@ -30,6 +28,8 @@ const (
 	ReadsFromNewSchemaVersion uint = 4
 	// NewSchemaOnlyVersion Lowest version that stops dual-writes in preparation to drop old schema
 	NewSchemaOnlyVersion uint = 6
+	// MinimumSchemaVersion Lowest version of database schema current code is able to handle
+	MinimumSchemaVersion uint = 13
```
Comment (alecbert-atl) [inline pkg/storage/schema.go:32]: On our BB side @ensary notes that we can't rollback once we migrate to schema version 15. Is this the same thing as needing our minimum schema version to be 15? I'm not clear on what happens if we migrate an environment to step up to 15 and then do a redeploy a week later, does this property being 13 mean on redeploy AI-API will start at 13?

Also I know this PR wasn't made to address it, but I see that we have properties around dual writes above, should these be removed now that release v0.40.0 of AI-API has taken those out?

Comment (gcase555) [reply pkg/storage/schema.go:32]: I should probably make this schema version 15 if we can't go backward from that

Comment (gcase555) [reply pkg/storage/schema.go]: ok so 15 fails the builds, looking into why now

Comment (dscgarcia) [reply pkg/storage/schema.go]: I think we need to also update the min version in main_test.go https://github.com/asecurityteam/asset-inventory-api/blob/master/integration/tests/main_test.go#L34, I didn't think about that with the PR to remove all of the old schema code. I'm surprised my PR went through, I didn't update this and tests still showed passing...

Comment (gcase555) [reply pkg/storage/schema.go]: Ill do that now

Comment (gcase555) [reply pkg/storage/schema.go:32]: we'll likely keep 13 here.

**Review by dscgarcia — APPROVED** [review]

**Review by akhalina — APPROVED** [review]

**Pull request merged**